integration-cli/docker_cli_commit_test.go

```diff
@@ -116,9 +116,9 @@ func (s *DockerSuite) TestCommitChange(c *check.C) {
 		"--change", "ENV test 1",
 		"--change", "ENV PATH /foo",
 		"--change", "LABEL foo bar",
-		"--change", "CMD [\"/bin/sh\"]",
+		"--change", `"CMD [""/bin/sh""]"`,
```

Comment:
@dnephin At the moment `--change` use `StringSliceVarP`. I searched the web and most of the same error message `bare " in non-quoted-field"` points to golang's `csv` package. I don't know enough detail about the implementation in cobra though.